src/components/v5/shared/UserAvatar/UserAvatar.tsx

```diff
@@ -44,6 +44,7 @@ const UserAvatar: FC<UserAvatarProps> = ({
           placeholderIcon="circle-person"
           seed={address && address.toLowerCase()}
           title={profile?.displayName || address || ''}
+          className={className}
```

Comment:
This prop is used in other place as well. I'm not sure if it's a good idea to reuse it here